shell/platform/windows/flutter_windows.cc

```diff
@@ -67,6 +67,9 @@ FlutterDesktopViewControllerRef FlutterDesktopCreateViewController(
           engine_properties.entry_point)) {
     return nullptr;
   }
+  
+  //Must happen after engine is running
```

Comment:
Missing period. Also, the PR needs a clang-format pass.

Reply:
Fixed.